### PR TITLE
[Editor] Add the possibility to move the caret with the keyboard once  an highlight has been made (bug 1881684)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by calixteman** [comment]
I forgot the test, let me add it.

File: src/display/editor/highlight.js [hunk 1]
```diff
@@ -530,6 +559,8 @@ class HighlightEditor extends AnnotationEditor {
     const div = super.render();
     if (this.#isFreeHighlight) {
       div.classList.add("free");
+    } else {
+      this.div.addEventListener("keydown", this.#boundKeydown);
```
Comment (Snuffleupagus) [inline src/display/editor/highlight.js:563]: This listener isn't removed anywhere, but I'm assuming that's actually *on purpose* in this case?

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

File: src/display/editor/highlight.js (hunk 1 above)
Comment (calixteman) [reply src/display/editor/highlight.js:563]: Why should we remove it ?

Comment (Snuffleupagus) [reply src/display/editor/highlight.js:563]: Recently I've reviewed a couple of PRs where some event listeners were accidentally not being removed as expected, hence I've simply amended by PR review "checklist" to look closer at any listeners :-)

**Comment by calixteman** [comment]
I had to disable the test for Chrome because I didn't to set the caret browsing mode.

**Pull request merged**